### PR TITLE
Support targetframewrok net5.0-windows.

### PR DESCRIPTION
**Pull request by angewa**
Support targetframewrok net5.0-windows.

### CODE REVIEW

**Comment by angewa** [comment]
@HongGit Please review this PR.

File: src/dotnet-svcutil/lib/src/Shared/MSBuildProj.cs [hunk 1]
```diff
@@ -159,6 +160,13 @@ public static async Task<MSBuildProj> ParseAsync(string projectText, string proj
                     var targetFramework = targetFrameworkElements.Last().Value.Trim();
                     if (!string.IsNullOrWhiteSpace(targetFramework))
                     {
+                        Regex re = new Regex(@"\w+\d+\.\d+");
```
Comment (HongGit) [inline src/dotnet-svcutil/lib/src/Shared/MSBuildProj.cs:163]: What is the reason of using regex here?  What about the approach with just comparing with "net5.0-windows*" approach?

**Review by HongGit — COMMENTED** [review]
Regex approach seems to be too broad, not narrow enough, thoughts?

File: src/dotnet-svcutil/lib/src/Shared/MSBuildProj.cs (hunk 1 above)
Comment (angewa) [reply src/dotnet-svcutil/lib/src/Shared/MSBuildProj.cs:163]: The possible target framework format examples are: netcoreapp3.1, net5.0, net5.0-windows, the regex here is to truncate the extra characters after version number, so for both net5.0 and net5.0-windows, or other target framework format like "net5.0***" appear in the furture, we will get net5.0 for the internal starter project to generate service proxy code.

Comment (angewa) [reply src/dotnet-svcutil/lib/src/Shared/MSBuildProj.cs]: > What is the reason of using regex here? What about the approach with just comparing with "net5.0-windows*" approach?

I have removed regex code and use "net5.0-windows" approach, could you please help review the update?

Comment (angewa) [reply src/dotnet-svcutil/lib/src/Shared/MSBuildProj.cs]: I've modified the code according to the meeting, could you please help review the update?

**Review by HongGit — APPROVED** [review]

**Comment by angewa** [comment]
@HongGit I have rebased the latest main code, the check now all passed.

**Pull request merged**